main/AndroidManifest.xml

```diff
@@ -192,24 +192,16 @@
             android:name=".AboutActivity"
             android:configChanges="keyboardHidden|orientation|screenSize"
             android:label="@string/about"
-            android:parentActivityName="cgeo.geocaching.MainActivity"
             android:windowSoftInputMode="stateHidden" >
-            <meta-data
-                android:name="android.support.PARENT_ACTIVITY"
-                android:value="cgeo.geocaching.MainActivity" />
```

Comment:
Removing the `android:parentActivityName` should go to a separate issue/PR, as it is not necessary for the `app id` PR, imho.